### ISSUE TITLE
Package manager terminology

### ISSUE THREAD

**Issue by tthiery** [dotnet/cli#1153]
Many package manager can modify their project.json and many do with same commands. E.g. npm install angular2 OR composer install foo/bar OR gem install maruku OR tsd install jquery.

Would not it be a good idea to expose nuget the same way like 'dotnet install System.Collection.Immutable'. Also a small research for other verbs like update etc would be cool. I think the current restore verb is not common for example.

Should be done before release ... Afterwards it may be too late :)

**Comment by tthiery** [comment]
Seems a duplicate of #76
